pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/deploy-custom-app/index.md

```diff
@@ -115,3 +115,7 @@ kubectl get helmreleases -n ${PROJECT_NAMESPACE}
 NAMESPACE               NAME            READY   STATUS                             AGE
 project-test-vjsfq      my-custom-app   True    Release reconciliation succeeded   7m3s
 ```
+
+## Upgrade custom applications
+
+You must maintain your custom applications manually. When upgrading DKP, ensure you validate any custom applications against the current version of Kubernetes for compatibility issues. When you are able, we recommend upgrading to the latest supported versions.
```

Comment:
@gracedo - Gaby mentioned this note, so I went ahead and added it to this PR. Please verify!